citations-tool/tool/src/java/org/sakaiproject/citation/tool/CitationHelperAction.java

```diff
@@ -2472,6 +2487,41 @@ public String buildNewResourcePanelContext(VelocityPortlet portlet, Context cont
 			contentProperties = new HashMap<String,Object>();
 		}
 		context.put("contentProperties", contentProperties);
+
+
+		BufferedReader streamReader = null;
+		try
+		{
+			ContentResource cr = contentService.getResource(MANAGING_LIBRARIES_JSON);
+			streamReader = new BufferedReader(new InputStreamReader(cr.streamContent(), "UTF-8"));
+			StringBuilder responseStrBuilder = new StringBuilder();
+
+			String inputStr;
+			while ((inputStr = streamReader.readLine()) != null){
+				responseStrBuilder.append(inputStr);
+			}
+			JSONObject jsonObject = JSONObject.fromObject(responseStrBuilder.toString());
+			JSONArray managingLibraries = jsonObject.getJSONArray("managingLibraries");
+			context.put("managingLibraries", managingLibraries);
+		} catch(JSONException e) {
+			logger.debug("JSONException reading managingLibraries.json for collection id: " + citationCollectionId, e);
```

Comment:
I'd make this a warning, but not include the exception, as it's something that the programmer probably doesn't need to take a look at, but someone like a sysadmin probably wants to see so they can get the file fixed.